### PR TITLE
feat(cli): ask user where to write the generated mnemonic

### LINKED ISSUE #160: CLI: Ask the user whether to write the generated mnemonic into a plain-text file
## Description

The title says it all.

## Motivation

Some users may not want that the generated mnemonic gets written to a plain-text file, which might become a security risk since not all users are able to safely delete files permanently in a way that is not recoverable.

## Are you planning to do it yourself in a pull request?

Yes.

### PR DESCRIPTION
**Pull request by Alex6323**
# Description of change

This PR gives the user the choice whether to write the mnemonic to a file or to print it to the console or do both.

## Links to any relevant issues

Closes #160 

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: cli/src/helper.rs [hunk 1]
```diff
@@ -51,3 +55,41 @@ pub async fn bytes_from_hex_or_file(hex: Option<String>, file: Option<String>) -
...
+    let mut file = OpenOptions::new().create(true).append(true).open(path).await?;
+    file.write_all(format!("{mnemonic}\n").as_bytes()).await?;
+
+    println_log_info!("IMPORTANT:");
+    println_log_info!(
+        "Mnemonic has been written to '{MNEMONIC_FILE_NAME}'. Make sure it resides in a secure location."
+    );
+    println_log_info!(
+        "It is the only way to recover your account if you ever forget your password and/or lose the stronghold file."
+    );
```
Comment (Thoralf-M) [inline cli/src/helper.rs]: This warning should also be there if it's only written to console

Comment (Alex6323) [reply cli/src/helper.rs]: alright :+1:

**Review by Thoralf-M — DISMISSED** [review]

File: cli/src/helper.rs [hunk 2]
```diff
@@ -51,3 +55,41 @@ pub async fn bytes_from_hex_or_file(hex: Option<String>, file: Option<String>) -
...
 }
+
+pub async fn get_mnemonic() -> Result<String, Error> {
+    let mnemonic = generate_mnemonic()?;
+    println_log_info!("Mnemonic has been generated.");
+
+    let choices = [
+        "Write To Console Only",
+        "Write To File Only",
+        "Write To Console and File",
```
Comment (thibault-martinez) [inline cli/src/helper.rs]: What's up with the weird capitalisation?

Comment (thibault-martinez) [reply cli/src/helper.rs]: I'd remove them, feels inconsistent

Comment (Alex6323) [reply cli/src/helper.rs]: It's actually called `https://en.wikipedia.org/wiki/Title_case`, but I messed it up a bit.

Comment (thibault-martinez) [reply cli/src/helper.rs]: Let's just do a normal sentence anyway

**Review by Thoralf-M — DISMISSED** [review]

File: cli/src/helper.rs [hunk 3]
```diff
@@ -51,3 +55,41 @@ pub async fn bytes_from_hex_or_file(hex: Option<String>, file: Option<String>) -
         None
     })
 }
+
+pub async fn get_mnemonic() -> Result<String, Error> {
```
Comment (thibault-martinez) [inline cli/src/helper.rs]: I feel like `get` is a bit off 🤔 
It assumes it already exists, I think I would also call it `generate_mnemonic` tbh

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**